taca/cleanup/cli.py

```diff
@@ -53,9 +53,12 @@ def milou(ctx, site, days, dry_run):
               help="Days to consider as thershold for removing analysis data")
 @click.option('--only_fastq', is_flag=True, help="Clean only fastq data in 'irma'")
 @click.option('--only_analysis', is_flag=True, help="Clean only analysis data in 'irma'")
-@click.option('-n','--dry_run', is_flag=True, help='Perform dry run i.e. Executes nothing but log')
+@click.option('--exclude_projects', type=click.STRING,
+              help="A project or a file with list of project to exclude from deleting, Both name or id can be given")
```

Comment:
in reality you accept a comma separated list of projects, so better to tell this in the help